libraries/ms-common/src/stm32f0xx/i2c.c

```diff
@@ -1,46 +1,46 @@
 #include "i2c.h"
 #include <stdbool.h>
-#include "stm32f0xx.h"
 #include "log.h"
+#include "stm32f0xx.h"
 
 // Arbitrary timeout
 #define I2C_TIMEOUT 1000000
-#define I2C_TIMEOUT_WHILE_FLAG(i2c, flag, status) \
-do { \
-  uint32_t timeout = (I2C_TIMEOUT); \
-  while (I2C_GetFlagStatus(i2c, flag) == status) { \
-    timeout--; \
-    if (timeout == 0) { \
-      LOG_DEBUG("Timeout: %d waiting for %d to change\n", flag, status); \
-      return STATUS_CODE_TIMEOUT; \
-    } \
-  } \
-} while (0)
-
-#define I2C_STOP(i2c) \
-do { \
-  I2C_TIMEOUT_WHILE_FLAG(i2c, I2C_FLAG_STOPF, RESET); \
-  I2C_ClearFlag(i2c, I2C_FLAG_STOPF); \
-} while (0)
+#define I2C_TIMEOUT_WHILE_FLAG(i2c, flag, status)                          \
+  do {                                                                     \
+    uint32_t timeout = (I2C_TIMEOUT);                                      \
+    while (I2C_GetFlagStatus(i2c, flag) == status) {                       \
+      timeout--;                                                           \
+      if (timeout == 0) {                                                  \
+        LOG_DEBUG("Timeout: %d waiting for %d to change\n", flag, status); \
+        return STATUS_CODE_TIMEOUT;                                        \
+      }                                                                    \
+    }                                                                      \
+  } while (0)
+
+#define I2C_STOP(i2c)                                   \
+  do {                                                  \
+    I2C_TIMEOUT_WHILE_FLAG(i2c, I2C_FLAG_STOPF, RESET); \
+    I2C_ClearFlag(i2c, I2C_FLAG_STOPF);                 \
+  } while (0)
 
 typedef struct {
   uint32_t periph;
   I2C_TypeDef *base;
 } I2CPortData;
 
-static I2CPortData s_port[NUM_I2C_PORTS] = {
-  [I2C_PORT_1] = { .periph = RCC_APB1Periph_I2C1, .base = I2C1 },
-  [I2C_PORT_2] = { .periph = RCC_APB1Periph_I2C2, .base = I2C2 }
-};
+static I2CPortData s_port[NUM_I2C_PORTS] = {[I2C_PORT_1] = {.periph = RCC_APB1Periph_I2C1,
```

Comment:
The spacing of `{.` on this line is a bug in clang-format that was fixed in late July it should be fixed with the next release.